### ISSUE TITLE
Question on iOS vs. Android

### ISSUE THREAD

**Issue by darkman088** [WhisperSystems/Signal-Android#5314]
Hello,

please apologize if my post is not appropriate here. I like Signal very much, I have installed it on both of my Phones - a Samsung phone on Android and on my iPhone. And I am really puzzled by the big differences in the app on both OS. I could even say that it almost looks like a different app. Here I would like to accent on 2 things:

1) Some time ago, I had raised a question, that while speaking on Signal on my Android phone, it is not possible to put the Signal call on hold, in order to take the phone call. I have got an answer, that it is not possible. On my iPhone though, I was surprised, that this works! I was speaking on Signal, I received a phone call and I was able to take the phone call !!

2) Security settings - I was pleased with the many security features offered on Android - it is possible to set a PIN code in order to access the app. I was also delighted about the option to set a number of messages to be kept, which I have set very low, so all the old messages simply go to /dev/zero
Both these options are not available on iOS :-/ Even when I enter the settings, they look almost empty, compared to the numerous settings and options on Android...

Can you please shortly comment on why it is like that ? Just asking out of curiosity... I hope that the app will further evolve on both platforms and will provide even more features!

Thank you for providing this magnificent software for free!! I wish you all the best for the future!

**Comment by JavaJens** [comment]
I think this is more of a support question. 
Anway, 1) What was the answer to the question? Maybe you can make it as a bug/feature request
2) IIRC this is due to the fact that iOS offers encryption natively, whereas the device encryption on Android is/was weak at best. Regarding auto-delete of messages I think you should open this as a bug/feature on Signal-iOS.

**Comment by darkman088** [comment]
Thank you, JavaJens, but you left a few things in your answer, on which I would need further clarification.
1) I have already made a bug/feature request and the answer was, that on Android it is not possible. But then I found out, that on iOS it's possible...
2) What is IIRC ? I would be glad to submit a feature request. Kindly let me know where can I do this... Providing the exact link and instructions.

Thank you for taking you time to deal with my questions and requests!
